bgcval2/functions/ice.py

```diff
@@ -0,0 +1,228 @@
+#
+# Copyright 2015, Plymouth Marine Laboratory
+#
+# This file is part of the bgc-val library.
+#
+# bgc-val is free software: you can redistribute it and/or modify it
+# under the terms of the Revised Berkeley Software Distribution (BSD) 3-clause license.
+
+# bgc-val is distributed in the hope that it will be useful, but
+# without any warranty; without even the implied warranty of merchantability
+# or fitness for a particular purpose. See the revised BSD license for more details.
+# You should have received a copy of the revised BSD license along with bgc-val.
+# If not, see <http://opensource.org/licenses/BSD-3-Clause>.
+#
+# Address:
+# Plymouth Marine Laboratory
+# Prospect Place, The Hoe
+# Plymouth, PL1 3DH, UK
+#
+# Email:
+# ledm@pml.ac.uk
+#
+"""
+.. module:: ice
+   :platform: Unix
+   :synopsis: This function calculates the ice fields.
+
+.. moduleauthor:: Lee de Mora <ledm@pml.ac.uk>
+
+"""
+
+import numpy as np
+from bgcval2.bgcvaltools.dataset import dataset
+
+
+tmask     = 0
+area    = 0
+lat    = 0
+loadedArea = False
+
+
+def loadDataMask(gridfn, maskname,):
+    global loadedArea
+    global tmask
+    global area
+    global lat
```

Comment:
here too - put globals up outside func environment

Reply:
Sorry about ice.py. I basically just copied and pasted it from the old version, without making any cuts or adaptations to modern styles. Will go over it and re-submit.